src/gluonts/model/transformer/_estimator.py

```diff
@@ -205,6 +220,11 @@ def __init__(
             self.prediction_length, self.config, prefix="dec_"
         )
 
+    def train(self, training_data: Dataset) -> Predictor:
+        self.input_repr.initialize_from_dataset(training_data)
+        self.output_repr.initialize_from_dataset(training_data)
+        return super().train(training_data)
```

Comment:
Same as above

Reply:
Seems like the Transformer does not have the data loader and validation stuff as of this moment. I added those and fixed the call to `super`.